src/libraries/System.Private.CoreLib/src/System/Threading/Timer.cs

```diff
@@ -171,6 +171,9 @@ private bool EnsureTimerFiresBy(uint requestedDuration)
         // need to look at the long list because the current time will be <= _currentAbsoluteThreshold.
         private const int ShortTimersThresholdMilliseconds = 333;
 
+        // Lock shared by the TimerQueue instances
```

Comment:
There's only one of these per TimerQueue, how about the following?
```suggestion
        // Lock shared by the TimerQueue and associated TimerQueueTimer instances
```